.bazelrc

```diff
@@ -14,4 +14,7 @@ build:linux --cxxopt=-std=c++20
 build:macos --cxxopt=-std=c++20 --cpu=darwin_x86_64
 build:windows --cxxopt=/std:c++20 --cxxopt=/Zc:preprocessor
 
+# emitting jdeps does not work when building the 2.0.0+ kotlin extractor
+build --@rules_kotlin//kotlin/settings:jvm_emit_jdeps=false
```

Comment:
didn't investigate it in depth, but I can look. It might maybe have to do with building the 2.0.0+ extractors with the 1.9 kotlin toolchain. Unfortunately it doesn't seem features-based, which means the only way to apply this to the 2.0.0+ extractors would be with a transition, that I'd really like to avoid.

Also, I don't think we have any need of jdeps.